src/app/globals.css

```
@@ -5,6 +5,11 @@
 :root {
   --font-span: "span", serif;
   --bw: 1440;
+
+  .swiper-slide-event-detail {
+    width: calc((325 / 1440) * 100 * 1vw) !important;
```

ini kenapa gak pakai variable --bw frans? untuk 1440nya?
```suggestion
    width: calc((325 /var(--bw)*100vw)) !important;
```